### ISSUE TITLE
UK airplanes on new UK Pacific carrier

### ISSUE THREAD

**Issue by limo49** [triplea-game/triplea#3458]
[test.zip](https://github.com/triplea-game/triplea/files/2100181/test.zip)

**My Operating System**: Windows 10
**Engine version**: 1.9.0.0.9687
**Map name and version**: World War II Global 1940 2nd Edition 3.9

**Can you describe how to trigger the error?**
- open savegame and start local game
- move the 2 UK tactical bombers from Western Australia to Sea Zone 39
- the move is forbidden with the msg "Not all air units can land"

**Instead of this error, what should have happened?**
- the 2 UK tac. bombers should move to SZ 39 because a new UK (Pacific) aircraft carrier has been bought

**Comment by beelee1** [comment]
I checked it out and couldn't make it work. tried a few different things. Thought it would have something to do with the split units for UK and UKP, with UKP not changing to UK until after end turn, but that wasn't it and as allies it shouldn't matter.

The original planes at game start will land on a new carrier. so yea, something aint right : )

**Comment by ron-murhammer** [comment]
@limo49 Yeah, the limitation is that the engine doesn't consider the newly built UK-Pacific carriers only UK carriers. I'll have to see how easy/hard it would be to add that check.

**Comment by ron-murhammer** [comment]
@limo49 PR to fix the issue #3460. I tested with your save and it works after that fix.

**Comment by limo49** [comment]
Thank you

**Comment by ron-murhammer** [comment]
@limo49 Should be able to test the fix with this pre-release: https://github.com/triplea-game/triplea/releases/tag/1.9.0.0.10375

**Comment by limo49** [comment]
I tested with 10375 and it seems working
